### ISSUE TITLE
Icon-request: icon-globe with other continents

### ISSUE THREAD

**Issue by maricato** [FortAwesome/Font-Awesome#281]
Hello!

The Font Awesome v2.0 is great, and it has finally the icon-globe! =)
However, the actual icon-globe represents America, and here in Europe and other Countries, I think the users would love the globe with the respective continent.

Could you draw other types of the icon-globe?

**Comment by hebbet** [comment]
+1

**Comment by T-net** [comment]
+1

**Comment by G-Driver** [comment]
+1 Euro

**Comment by Axxiss** [comment]
A continent (America, Asia, etc) centered version wouldn't be too much of a load.

Imho a globe per country be another separate project

+1

**Comment by tomahock** [comment]
+1 to Europe centered icon.

**Comment by eddiphant** [comment]
+1 to Europe centered icon.

**Comment by drammock** [comment]
Just noticed that the MathJax website has something North-Atlantic-centered: https://www.mathjax.org/img/browsers.svg

It weighs in around 12 kB, but running it through Inkscape's optimization defaults gets it down to about 7 kB. Further optimizations involving simplifying the actual paths (not just the SVG code) could bring it down even further.

**Comment by eschatonic** [comment]
+1 for a Africa/Eurasia-centric version, similar to Facebook's

**Comment by skechboy** [comment]
+1

**Comment by codersknowwhy** [comment]
+1

**Comment by dahamsta** [comment]
+1 for europe, but the MathJax style one referenced by @drammock would work too.

**Comment by eggleton** [comment]
+1

**Comment by kepi** [comment]
+1

**Comment by kepi** [comment]
:+1:

**Comment by greq7** [comment]
+1 for europe

**Comment by valioDOTch** [comment]
+1 for the "global" solution with points simplified (adding in on *hugoroy*)

**Comment by marczking** [comment]
+1

**Comment by brylie** [comment]
Here is an svg showing Africa, Europe, and some of Asia:

![Globe showing Africa, Asia, and Europe](https://openclipart.org/image/400px/svg_to_png/236045/globe-europe-simple-bw.png)

The source is available on OpenClipArt:
* [Simple globe facing Europe and Africa](https://openclipart.org/detail/236045/simple-globe-showing-africa-asia-and-europe-in-black-and-white-detailed)

**Comment by G-Driver** [comment]
+1

**Comment by maxlath** [comment]
+1 for an Africa-Eurasia globe icon without too much precision to keep it light (cf @hugoroy comment)

**Comment by michaelhodgins** [comment]
+1

**Comment by jbnv** [comment]
+1

**Comment by sanek4life** [comment]
+1

**Comment by vipersec** [comment]
I would also love to have a localized globe.

**Comment by matinfo** [comment]
Adding just 2 globe icons, one for europe-afriqua and russia-asia, seams no to mutch weight, and this will make this web-font internationally really Awesome, no?

**Comment by caub** [comment]
https://pixabay.com/static/uploads/photo/2015/07/31/09/48/globe-868846_960_720.jpg doesn't completely fit in a circle, but shows the whole globe

**Comment by yospyn** [comment]
+1 for what matinfo said, two rotations such as fa-globe-africa and fa-globe-asia would be lovely.  Or keep it simple such as Africa w/western Asia is lovely also.  These would be very useful for overseas but also US-based design regarding international development.  They frown on the north american facing globe, can't use it.  Also FA is probably the most useful web design tool I've used in the past two years, so quite appreciative.

**Comment by caub** [comment]
I don't like the idea to have 3 world sections, the idea is to represent the whole world as one icon, not continents, you can still have fa-america, fa-europe, fa-asia, then you could want to have fa-america-europe on the same icon, ... 

http://geospatial.blogs.com/.a/6a00d83476d35153ef0154365475cd970c-320wi,  http://2.bp.blogspot.com/-QAYfv4g0U64/UHlEhTQTBkI/AAAAAAAAAo0/J73NeyD_3H0/s1600/Global+position+tropical+rainforests+map.jpg makes more sense to me

**Comment by am12976** [comment]
An unbiased and neutral globe would be more appropriate. Please note below:

![globe-icons](https://cloud.githubusercontent.com/assets/20721882/17244064/2ad23e7e-5595-11e6-95e6-d5286fa854a7.jpg)

**Comment by hellsing71** [comment]
+1 for EU/Africa
+1 for Asia
+1 for the neutral version

I agree a simplified version would be good enough for webdesign projects. For poster printers there are several other solutions.

**Comment by bilo1967** [comment]
The best would be a **spinning** globe.

+1 for EU/Africa
+1 for Asia

**Comment by Leif-W** [comment]
+1 for geo-neutral as minimal stop-gap solution

+1 for a separate "internationalized" glyph package, with globes and flags, etc

+1 for EU
+1 for Asia
+1 for Africa
+1 for South America
+1 for Oceania
+1 for North Atlantic
+1 for Antarctic

+1 for spinning globe

I question the validity of the decision to keep things geocentric to North America at the exclusion of all others.  The reason that it makes the file "too big" seems like an excuse.  The v4.7 SVG file is >444,379 bytes.  The North American globe adds 3,803 (though could perhaps be less detailed, maybe get it to 3,000 bytes?), which is less than 0.9%.

To consider adding 3-4 more views, would increase the file size by less than 5%, 6-8 more views, less than 10%.  5-10% file size increase for 3-8 glyphs, though it seems like a small increase, is significant in comparison to the total number of glyphs provided (675?), the average which seems to use about 300-600 bytes (though many use 500-1000, and 1250-1500 is not uncommon).

But, to add 41 new glyphs and advertise it proudly like it's nothing, yet for years and years refuse to make the globe non-geocentric or give multiple options, is like a slap in the face.  It projects an attitude of egotistical dominance and laziness.  I am from America, and have no desire to send that message to users from other countries.  A pragmatic compromise is the geo-neutral "normalized" globe glyph, though some groups rather dislike that as well.

**Comment by Crissov** [comment]
Related Unicode emojis: #222

- [🌍 U+1F30D](http://emojipedia.org/earth-globe-europe-africa/)
- [🌎 U+1F30E](http://emojipedia.org/earth-globe-americas/)
- [🌏 U+1F30F](http://emojipedia.org/earth-globe-asia-australia/)
- [🌐 U+1F310](http://emojipedia.org/globe-with-meridians/)

**Comment by jtaglione** [comment]
Fontawesome is great, it is almost a life saver in many web projects. However, to see that this issue is opened since 5 years ago is a bit sad. USA < World, even on the internet. Please add at least fa-globe-eu-africa and fa-globe-asia and make a lot of people happy :) You seem to have SVGs ready, and people eager to give it to you!

**Comment by timoto** [comment]
I agree with jtaglione and all others supporting this non-US centric view of the world.

When there is such little response on such issues, one begins to wonder if making a comment is a worthwhile or completely pointless activity.

**Comment by AvengerDr** [comment]
Another hopeful request for a Europe-centric globe.

**Comment by nuflowx** [comment]
> Another hopeful request for a Europe-centric globe.

lol at being hopeful after more than half a decade of this being requested xD :P

**Comment by Leif-W** [comment]
I've completely abandoned FontAwesome and the entire technique of using fonts this way.  I use SVG based fonts now, which allow custom coloring of individual sections, and don't require complex tools like FontForge to manipulate.

**Comment by redoper1** [comment]
I think best would be to center it to the 0,0 GPS coordinates. There wouldn't be Asia, but it would be quite neutral.

**Comment by dejudicibus** [comment]
Currently there is only one globe icon, showing the North America continent.
I propose to extend globe icons to:
fa-globe-na-sa (Americas)
![image](https://user-images.githubusercontent.com/1431440/34941613-c74a1d76-f9f4-11e7-95e3-eff6769c7c73.png)
fa-globe-eu-af (Europe & Africa)
![image](https://user-images.githubusercontent.com/1431440/34941628-d98efa9c-f9f4-11e7-8bc1-0db672bb3c08.png)
fa-globe-as-oc (Asia and Oceania)
![image](https://user-images.githubusercontent.com/1431440/34941650-ea1b3a38-f9f4-11e7-84f8-14e01a48ccd2.png)
fa-globe-ao (Atlantic Ocean)
![image](https://user-images.githubusercontent.com/1431440/34941667-fd1c5fea-f9f4-11e7-954c-d2ad661dbce9.png)
fa-globe-po (Pacific Ocean)
![image](https://user-images.githubusercontent.com/1431440/34942008-55a14ea4-f9f6-11e7-977d-4e7a09b0fd47.png)
fa-globe-an (Antarctic)
![image](https://user-images.githubusercontent.com/1431440/34942286-60f635ac-f9f7-11e7-8dbf-6cdc7719d800.png)
fa-globe-wired (Wireframe)
![image](https://user-images.githubusercontent.com/1431440/34941746-599c9550-f9f5-11e7-9764-b4564dff6b42.png)
Thank you in advance.

**Comment by kayleighbray** [comment]
+1 for bringing back the more realistic globe (with less detail, as mentioned above)
New one is not as easily recognizable as the old globe was

**Comment by gerthermel** [comment]
+1 for old globe, one that looked more like earth. New globe, one with stripes, reminds me more like a basketball.

**Comment by sensibleworld** [comment]
https://fontawesome.com/icons/globe-asia?style=solid
https://fontawesome.com/icons/globe-americas?style=solid
https://fontawesome.com/icons/globe-africa?style=solid

**Comment by alitedi** [comment]
+1 For all Seven continents : Africa, Antarctica, Asia, Australia/Oceania, Europe, North America, and South America
